NSSNamespace/Initial_Bangazon_Site
# added supbtypes to ERD

## Changes
Changed ERD to now encompass product subtypes
- ```Product``` now has a ```ProductSubtype```, as opposed to a ```ProductType```.
- ```ProductSubType``` now has a relationship with ```Product``` and with ```ProductType```.

## To Test
- ```git fetch```
- ```git checkout origin/ew-updateERD```
- Open the bangazonERD.pdf file in the ERD folder, located in the root directory.  Make sure it reflects the relationships noted above.
